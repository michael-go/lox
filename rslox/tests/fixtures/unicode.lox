// סבבה
print 1 / 2;