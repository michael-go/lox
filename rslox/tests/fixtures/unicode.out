# exit code: 0
# stdout:
0.5

# stderr:

